### DISCUSSION TITLE
Vanilo demo app

### DISCUSSION THREAD

**Discussion by WMspace** [vanilophp/framework#186] [General]
Hi, can you please tell me if there are any plans to change or improve the Vanilo demo app?

**Comment by fulopattila122** [comment]
It's currently sitting rather in maintenance mode; we plan to replace it with a new, comprehensive application soon.

What would your requirements be? What would you need the most?

**Reply by nuovo10** [reply]
Hello
When do you plan to implement the new version?

Great job with Vanilo
Regards

**Comment by WMspace** [comment]
Hello @fulopattila122 ,
Thank you for taking the time to respond.
I'm just starting to get acquainted with Vanilo, but I can already tell that you've done a very good and much needed job.
When choosing an e-commerce framework, people look for different reviews and comparisons, so to be able to install and look at a future prototype is very cool.
A good demo is not only an example of good practices of using the framework, but also a great advertisement for it.
Now I am creating a project for accepting payments using payment cards of international systems, and then I plan to expand it to an online store.

It is interesting to see the Demo:
- accepting payments using different payment gateways;
- placing an order;
- creating a product grid in the frontend;
- product comparison;
- create and manage a wishlist;
- creation of a multivendor platform (Marketplace);
- import/export of goods.

Of course, this is too much for a simple demo version, but what can you do, they have such desires 😀.
It would also be very interesting to see a Vanilo server on Discord, it seems to me that Telegram is not a very convenient place to communicate, and it is also very acidic.

**Reply by fulopattila122** [reply]
Hi @iWpress 

Thank you for the feedback. Some items on your list are covered in the demo app:

- accepting payments using different payment gateways; :white_check_mark: [here](https://github.com/vanilophp/demo/blob/master/app/Http/Controllers/CheckoutController.php#L63), [here](https://github.com/vanilophp/demo/blob/master/app/Http/Controllers/MollieController.php) and [here](https://github.com/vanilophp/demo/blob/master/app/Http/Controllers/PaypalReturnController.php)
- placing an order:  :white_check_mark: [here](https://github.com/vanilophp/demo/blob/master/app/Http/Controllers/CheckoutController.php#L57)
- creating a product grid in the frontend: :white_check_mark:  [here](https://github.com/vanilophp/demo/blob/master/app/Http/Controllers/ProductController.php#L20) and [here](https://github.com/vanilophp/demo/blob/master/resources/views/product/index.blade.php)
- product comparison: :x: - this can be built by listing side-by-side product [attributes](https://vanilo.io/docs/4.x/products#all-product-fields) and [properties](https://vanilo.io/docs/4.x/properties)
- create and manage a wishlist; :x: - vanilo has no built-in wishlist feature
- creation of a multivendor platform (Marketplace); :x: vanilo has no multivendor feature at the moment
- import/export of goods: :heavy_check_mark:   the only import plugin for the open source Vanilo is this: https://github.com/vanilophp/woocommerce-sync

Besides Github, we also started to collect feature requests here: https://vanilo.canny.io/feature-requests
So feel free to add requests on that page (one feature per post).
Once you have submitted the feature request, other users will be able to upvote it, allowing us to prioritize features based on demand.

**Comment by WMspace** [comment]
Thank you very much!
I ran to install 😀.
You are the best!
